script/testnet/deploy/L1FactoryDeployAndSetup.s.sol

```diff
@@ -7,16 +7,16 @@ import {console} from 'forge-std/Test.sol';
 import {IL1OpUSDCFactory} from 'interfaces/IL1OpUSDCFactory.sol';
 
 contract L1FactoryDeployAndSetup is Script {
-  address public constant USDC = 0x1c7D4B196Cb0C7B01d743Fbc6116a902379C7238;
-  address public deployer = vm.rememberKey(vm.envUint('SEPOLIA_PK'));
+  address public deployer = vm.rememberKey(vm.envUint('TESTNET_PK'));
+  address public usdc = vm.envAddress('TESTNET_USDC_IMPLEMENTATION');
 
   function run() public {
+    vm.createSelectFork(vm.rpcUrl(vm.envString('TESTNET_RPC')));
```

Comment:
is this line correct? We need to fork? Mainnet L1FactoryDeployAndSetup does not have this line

Reply:
Needs removal, good catch